<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPreferences</class>
 <widget class="QDialog" name="DlgPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QListWidget" name="lwList">
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/Preferences.ico</normaloff>:/Icons/Preferences.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Appearance</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/Appearance.ico</normaloff>:/Icons/Appearance.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Folders &amp; Locations</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/FileOpen.ico</normaloff>:/Icons/FileOpen.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plugins</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/Plugin.ico</normaloff>:/Icons/Plugin.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Workspaces</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/Workspaces.ico</normaloff>:/Icons/Workspaces.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Miscellaneous</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/General.ico</normaloff>:/Icons/General.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Info</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/info.ico</normaloff>:/Icons/info.ico</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/Icons/Bug.ico</normaloff>:/Icons/Bug.ico</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lTitleIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lTitle">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Preferences</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="swContent">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pAppearance">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="lAppearance">
             <property name="text">
              <string>Appearance:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbAppearance">
             <item>
              <property name="text">
               <string>System Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Windows XP Style</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fusion</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pFolders">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QListWidget" name="lwLocationCategories"/>
           </item>
           <item>
            <widget class="QStackedWidget" name="swLocationSelector">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="pSingleSelection">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="lLocationSingleFolder">
                 <property name="text">
                  <string>Folder:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLineEdit" name="leLocationSingleFolder">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbBrowseLocationSingleFolder">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../Resources.qrc">
                     <normaloff>:/Icons/FileOpen.ico</normaloff>:/Icons/FileOpen.ico</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>417</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="pMultipleSelection">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="lLocationMultipleFolders">
                 <property name="text">
                  <string>Folders:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QListWidget" name="lwLocationMultipleFolders"/>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QPushButton" name="pbLocationMultipleFoldersAdd">
                     <property name="text">
                      <string>Add</string>
                     </property>
                     <property name="icon">
                      <iconset resource="../Resources.qrc">
                       <normaloff>:/Icons/Add.ico</normaloff>:/Icons/Add.ico</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pbLocationMultipleFoldersRemove">
                     <property name="text">
                      <string>Remove</string>
                     </property>
                     <property name="icon">
                      <iconset resource="../Resources.qrc">
                       <normaloff>:/Icons/Remove.ico</normaloff>:/Icons/Remove.ico</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pPlugins">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>The plugin interface isn't available now!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pWorkspaces"/>
         <widget class="QWidget" name="pMiscellaneous"/>
         <widget class="QWidget" name="pInfo"/>
         <widget class="QWidget" name="pDebug"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbReset">
       <property name="text">
        <string>Reset Preferences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
